cyraft/node.py

```diff
@@ -175,7 +175,11 @@ async def _start_election(self) -> None:
         for remote_node in self.cluster:
             if remote_node._node.id != self._node.id:
                 _logger.info("Node ID: %d -- Sending request vote to node %d", self._node.id, remote_node._node.id)
-                response = await remote_node.serve_request_vote_impl(request, self._node.id)
+                # response = await remote_node._serve_request_vote_impl(request, self._node.id)
+                client_node = self._node.make_client(
+                    sirius_cyber_corp.RequestVote_1, remote_node._node.id, "request_vote"
+                )
+                response, _ = await client_node.call(request)
```

Comment:
This is type-incorrect because this method returns None on timeout (if there's no response); the return type is `None | tuple[ServiceResponseType, pycyphal.transportTransferFrom]` (the latter contains metadata from the transport layer, like the transfer-ID, timestamps, and such). If you get None, it will blow up at the tuple unpacking statement. There are two correct solutions:

- Store the result into a temporary, check if it is a non-None, then unpack. This is very verbose and it looks like you don't want the metadata anyway.
- Do not use the verbose `call` method, call the client itself. This way you get the simpler return type: `None | ServiceResponseType`. See https://pycyphal.readthedocs.io/en/stable/api/pycyphal.presentation.html#pycyphal.presentation.Client.__call__

```suggestion
                response = await client_node(request)
```